tests/php/Model/SiteTreeTest.php

```diff
@@ -2060,12 +2057,27 @@ public function provideSanitiseExtraMeta(): array
                 '<link rel="canonical" accesskey="X" onclick="alert(1)" name="x" />',
                 '<link rel="canonical" name="x">',
                 'Multiple attributes are removed'
-            ],
-            [
-                '<link rel="canonical" href="valid" ;;// somethingdodgy < onmouseover=alert(1)',
+            ]
+        ];
+    }
+
+    public function testSanatiseInvalidExtraMeta()
+    {
+        $siteTree = new SiteTree();
+        $siteTree->ExtraMeta = '<link rel="canonical" href="valid" ;;// somethingdodgy < onmouseover=alert(1)';
+        if (class_exists(HTML5Value::class)) {
```

Comment:
`HTML5Value` could exists without being the default.

Reply:
It makes itself the default - https://github.com/silverstripe/silverstripe-html5/blob/2/_config/html.yml#L6

The only scenario where it's not the default is if someone installs silverstripe/html5 and then manually configures to use HTML4Value ... which is not ever going to be an issue in CI

Reply:
Have updated